data/rim jobs.txt

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Southbound Shipment [0]"
+	name "Shipment to <planet>"
+	job
+	repeat
+	description "Deliver <cargo> to <destination>, the home of Southbound Shipyards, the largest ship manufacturer in the region. Payment is <payment>."
+	cargo "weapons" 5 2 .1
+	to offer
+		random < 50
```

Comment:
These will stack up real quick at 50%, and with them all going to the same location, makes for nice job strategy.

Reply:
Indeed; these should probably have lower frequencies.